### PR TITLE
remove minor bug and made plugin compatible for release 2021.3

### PR DESCRIPTION
**Pull request by homburgs**

### CODE REVIEW

File: src/main/java/gitflow/actions/FinishBugfixAction.java [hunk 1]
```diff
@@ -62,12 +62,12 @@ public void runAction(final Project project, final String bugfixName){
...
             @Override
             public void run(@NotNull ProgressIndicator indicator) {
-                GitCommandResult result =  myGitflow.finishBugfix(myRepo, bugfixName, errorLineHandler);
-
                 //get the base branch for this bugfix
                 final String baseBranch = gitflowConfigUtil.getBaseBranch(branchUtil.getPrefixBugfix()+bugfixName);
 
-                if (result.success()) {
+				GitCommandResult result =  myGitflow.finishBugfix(myRepo, bugfixName, errorLineHandler);
+
```
Comment (OpherV) [inline src/main/java/gitflow/actions/FinishBugfixAction.java:69]: Please fix the white space issues

**Review by OpherV — CHANGES_REQUESTED** [review]
Thanks for the PR.
Please address the minor issues in the bugfix bug fix and remove the change in the gradle config. 

Version compatibility is a different issue that I'm tackling separately

**Comment by medbenmakhlouf** [comment]
what is the timeline to merge this please ? thanks

**Comment by banagale** [comment]
@medbenmakhlouf I built the project [reproducing @homburgs](https://github.com/banagale/gitflow4idea/commit/65fdc1109fd428453842f0f34d70f5412e1329e8) #343 commits.

It runs in Pycharm 2021.3 RC2 and the gold, 2021.3. I'm using it in Pycharm `Build #PY-213.5744.248, built on November 29, 2021`

If someone wants to do this on their own you have to build the .jar file with intellij ultimate. Instructions [in this issue](https://github.com/OpherV/gitflow4idea/issues/241).

Or download and unzip the following:

* [Download - gitflow4idea-0.7.7.jar.zip](https://github.com/OpherV/gitflow4idea/files/7655412/gitflow4idea-0.7.7.jar.zip)

Instructions to manually install this or your own jar build for pycharm [are here.](https://www.jetbrains.com/help/pycharm/2021.3/managing-plugins.html#install_plugin_from_disk)

There are good reasons opherV does not rush releases. 

While I'm using this successfully myself, use this at your own risk. :)

See below for minor possible bug:

**Comment by banagale** [comment]
Can anyone duplicate a bug I'm seeing in the [manual temporary build](https://github.com/OpherV/gitflow4idea/pull/343#issuecomment-986180937) I posted above?  

After installing the plugin manually, and restarting Pycharm, I was no longer seeing git flow in the toolbar (as though it was not installed) and it was not in the "installed" or "marketplace" plugins list when searching for "git flow". 

I suspect it is a bug, but could be because I didn't update the plugin version number or install file as I roughshod threw together this temporary patch? 

The reason I noticed is that gitflow seemed to disappear. Then when I went to manually install it again, and restart. It reappeared in the toolbar, and also in the plugins Installed list.

File: src/main/java/gitflow/actions/FinishBugfixAction.java (hunk 1 above)
Comment (CC007) [reply src/main/java/gitflow/actions/FinishBugfixAction.java:69]: considering that there was no activity on this PR, I think you'd have to change that yourself.

**Pull request closed without merging**